src/purejavahidapi/windows/HidDevice.java

```diff
@@ -189,6 +180,13 @@ synchronized public int setOutputReport(byte reportID, byte[] data, int length)
 		m_OutputReportMemory.write(1, data, 0, length);
 
 		if (!m_ForceControlOutput) {
+			// bManualReset parameter has to be set to true to work with WaitForSingleObject
+			m_OutputReportOverlapped.hEvent = CreateEvent(null, true, false, null);
```

Comment:
A corresponding `CloseHandle` is needed for `CreateEvent`.  This Create/Close pair can happen once (when the OVERLAPPED is created / no longer needed), instead of on each write cycle.

Reply:
Done.